mathics/builtin/box/graphics3d.py

```diff
@@ -727,14 +727,7 @@ def init(self, graphics, style, item):
         if len(item.leaves) != 1:
             raise BoxConstructError
 
-        points = item.leaves[0].to_python()
-        if not all(
-            len(point) == 3 and all(isinstance(p, numbers.Real) for p in point)
-            for point in points
-        ):
-            raise BoxConstructError
-
-        self.points = [Coords3D(graphics, pos=point) for point in points]
+        self.points = [Coords3D(graphics, pos=point) for point in item.leaves[0]]
```

Comment:
It feels like something is wrong here in having to propagate `graphics` down to each point rather than storing it once in say self.

Reply:
I am wondering if we we shouldn't rename the `Coords3D` and `Coord` to something like `GraphicsCoords3D` and `GraphicsCoord2D`  or maybe just one `GraphicsCoord` with a dimension. to distinguish the fact that this kind of coordinate belongs in a some sort of graphics and therefore has a `graphics` field and it is meaningful for this kind of point to be scaled and for example. 

The new coordinate class in contrast, does not care whether its is placed in a graphic image or not.